packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -191,11 +186,6 @@ export default function QueryInspectorControls( props ) {
 
 	return (
 		<>
-			{ !! postType && (
-				<BlockInfo>
-					<CreateNewPostLink postType={ postType } />
```

Comment:
>as I believe removing it would require proper documentation or mentioning it in the changelog.

We also need to remove this component. It's only used in Query Loop and not exported separately. No need to update anything else, like changelog or doc.